roles/update-host/README.md

```diff
@@ -0,0 +1,47 @@
+# update-hosts role
+
+This role drive when the role updates the packages on the inventory
```

Comment:
Change this to "This role performs an updates and reboots a host (and waits for it to come back)".